### PR TITLE
GUT-138: Story module updates

### PR DESCRIPTION
**Pull request by ethitter**
### Doneness Checklist:

Pre-release # (or n/a):

- [x] Updated changelog with summary of changes under `Unpublished` section
- [x] `npm run prod` in this repo outputs no changes
- [x] Tested changes in pmc-spark [via a pre-release](https://confluence.pmcdev.io/x/XhOeAw):
- - [ ] If changes to build tools: npm scripts `prod`, `lint`, and `dev` scripts run as expected
- - [ ] If changes to Larva server: Static site generates as expected in a theme  (avail. on a URL {brand}.stg.larva.pmcdev.io)

### CODE REVIEW

File: packages/larva-patterns/modules/story/story.center-vertical.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const clonedeep = require( 'lodash.clonedeep' );
+
+const story_center_vertical = clonedeep( require( './story.vertical' ) );
+
+const horizontal_alignment = ' lrv-u-justify-content-center';
```
Comment (laras126) [inline packages/larva-patterns/modules/story/story.center-vertical.js:5]: These controller files should be kept as simple as possible - someday we will decide on a better syntax for them, and can create codemods or similar to replace with the new syntax. For that reason, add classes directly in the assignment to the object property vs. controlling redundancy with a variable, to keep the syntax as simple and consistent as possible.

A linter for these files to automate rules such as this that are currently in my brain only is part of the Larva Rollout, and now has a ticket - https://jira.pmcdev.io/browse/GUT-186 – but will be pretty low priority.

File: packages/larva-patterns/modules/story/story.right.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+const clonedeep = require( 'lodash.clonedeep' );
+
+const story_right = clonedeep( require( './story.prototype' ) );
+
+const horizontal_alignment = ' lrv-u-justify-content-end';
```
Comment (laras126) [inline packages/larva-patterns/modules/story/story.right.js:5]: No variable - write the class in line.

File: packages/larva-patterns/modules/story/story.right.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+story_right.story_grid_primary_classes = story_right.story_grid_primary_classes.replace(
```
Comment (laras126) [inline packages/larva-patterns/modules/story/story.right.js:7]: Copy/paste to reassign the entire value with the replaced string - no `.replace`. Allowing the use of array and string manipulation methods resulted in the incredible maze that is known as the VY code-base. It's redundant, yes, but definitely preferable to that.

File: packages/larva-patterns/modules/story/story.vertical.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+const clonedeep = require( 'lodash.clonedeep' );
+
+const story_vertical = clonedeep( require( './story.prototype' ) );
+
+story_vertical.story_grid_classes = story_vertical.story_grid_classes.replace(
```
Comment (laras126) [inline packages/larva-patterns/modules/story/story.vertical.js:5]: No `.replace` in this file. 

This is a method we would disallow in the linter.

**Review by laras126 — CHANGES_REQUESTED** [review]
A few syntax updates required in the controller files – we don't have linting for these yet, but will someday.

**Comment by ethitter** [comment]
@laras126 - thanks for the feedback! This is ready for another review

**Comment by laras126** [comment]
@ethitter Code looks good but please check the Doneness checklist! Really just the CHANGELOG one is relevant here.

**Comment by ethitter** [comment]
@laras126 - updates made, this is ready for another look

**Review by laras126 — APPROVED** [review]
All good, thanks!

**Pull request merged**